web-ui/src/main/resources/catalog/components/viewer/owscontext/OwsContextService.js

```diff
@@ -446,6 +446,14 @@
           resourceList.layer.push(params);
         });
 
+        function getNonProxifiedUrl(url) {
```

Comment:
I added similar code in this PR: https://github.com/geonetwork/core-geonetwork/pull/3092/files, it uses also `gnGlobalSettings.proxyUrl` instead of hardcoded value.

Maybe we should check to move it to some utilitity class?

Reply:
I agree with Jose. At least, don't use the hardcoded value. But better if we use some kind of utility.